scripts/common/setup_env.sh

```diff
@@ -6,30 +6,30 @@
 set -e
 
 if [ -z "$1" ]; then
-    echo "- Missing mandatory argument: IPDK_RECIPE"
-    echo " - Usage: source setup_env.sh <IPDK_RECIPE> <SDE_INSTALL> <DEPEND_INSTALL>"
+    echo "- Missing mandatory argument: SDE_INSTALL"
+    echo " - Usage: source setup_env.sh <SDE_INSTALL> <P4CP_DEP_INSTALL> <P4CP_INSTALL>"
     return 0
 fi
 
 if [ -z "$2" ]; then
-    echo "- Missing mandatory argument: SDE_INSTALL"
-    echo " - Usage: source setup_env.sh <IPDK_RECIPE> <SDE_INSTALL> <DEPEND_INSTALL>"
+    echo "- Missing mandatory argument: P4CP_DEP_INSTALL"
```

Comment:
You may not rename `DEPEND_INSTALL`. Like `SDE_INSTALL`, this environment variable has internal significance in both the CMake build files and the helper scripts.

Reply:
Okay, will keep it as DEPEND_INSTALL for now. We should consider changing it since DEPEND_INSTALL is very generic and doesn't communicate that these are P4 Control plane dependencies. Now, we are seeing use cases where only bf_switchd can be used as runtime application. In this case, these dependencies are not required

Reply:
Actually, most of them are _Stratum_ dependencies. I've started using the latter term in documentation.